kusari/cmd/configuration.go

```diff
@@ -0,0 +1,15 @@
+package cmd
+
+import "github.com/spf13/cobra"
+
+func KusariConfiguration() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "configuration",
+		Short: "Configuration",
```

Comment:
We should change the short and long descriptions to be more descriptive - what are we configuring?